falcon/app.py

```diff
@@ -43,10 +44,10 @@
 ])
 
 
-class API:
+class App:
```

Comment:
There is an autodoc for this class in the docs and a few of the examples use `api.add_static_route` or `api.router_options`. Would you change `api` to `app`?

Reply:
Yes, I'll change it for consistency